client/package-lock.json

```diff
@@ -0,0 +1,2273 @@
+{
```

Comment:
should this file be ignored?

Reply:
`package.json` alone specifies a range of versions to pull, if the file isn't included it's possible a newer version of a dependency can be pulled when CI is ran that has an issue that breaks something, leading to an issue that happens only in deploy but not in local development.

Reply:
That being said, I understand it makes commit diffs large and the issue is an edge case. If you think I should remove them I will.